packages/button/src/ButtonBase.ts

```diff
@@ -32,6 +32,10 @@ import buttonStyles from './button-base.css.js';
 export class ButtonBase extends ObserveSlotText(LikeAnchor(Focusable), '', [
     'sp-overlay,sp-tooltip',
 ]) {
+    spectrumConfig = {
+        downstate: true,
```

Comment:
We likely want the value here not to be a boolean but an array of values that could include the current "system", e.g.:
```suggestion
        downstate: ['spectrum2'],
```

Reply:
This way, some functionality can be delivered a la:
```
    downstate: ['spectrum1', 'spectrum2']
```

We should work to include these values an enums, to simplify sharing and consumption.

Reply:
Great idea! I'll update it accordingly